src/core/xfa/parser.js

```diff
@@ -14,6 +14,7 @@
  */
 
 import {
+  $acceptWhites,
```

Comment:
I assume whites = whitespace? If so, let's change to full word here and elsewhere.